doc/integration.md

```diff
@@ -33,10 +33,21 @@ dependencies {
 }
 ```
 
+###Granting internet access
+To function properly, the SDK requires access to the internet. You will need to make sure that the following user permission appears in your application's manifest:
+
+```xml
+<manifest xlmns:android="http://schemas.android.com/apk/res/android">
+  ...
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
I think  Best usage is add all permissions needed to Sample app and show entire project with relevant permissions.